compiler/codegen.go

```diff
@@ -157,6 +157,13 @@ func (n *varDeclTupleNode) Codegen(ostream io.Writer) {
 	fmt.Fprintf(ostream, "store %d\n", info.address)
 	info, _ = n.ctx.lookup(n.high)
 	fmt.Fprintf(ostream, "store %d\n", info.address)
+	if n.nodeName == "divmodw" {
+		info, _ = n.ctx.lookup(n.rlow)
+		fmt.Fprintf(ostream, "store %d\n", info.address)
+		info, _ = n.ctx.lookup(n.rhigh)
+		fmt.Fprintf(ostream, "store %d\n", info.address)
+	}
+
```

Comment:
it looks like would be better to define `varDeclQuadrupleNode` and do not have `divmodw` mention here